shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -1143,10 +1091,10 @@ \subsection{Rewards Distribution Calculation}
   \emph{Calculation to reward a single stake pool}
   %
   \begin{align*}
-      & \fun{rewardOnePool} \in \PParams \to \Coin \to \N \to \HashKey \to \PoolParam\\
-      & ~~~\to \Stake \to \Avgs \to \Coin \to \powerset{\AddrRWD}
+    & \fun{rewardOnePool} \in \PParams \to \Coin \to \N \to \N \to \HashKey \to \PoolParam\\
+      & ~~~\to \Stake \to \Coin \to \powerset{\AddrRWD}
            \to (\AddrRWD \mapsto \Coin)\times\Coin \\
-      & \rewardOnePool{pp}{R}{n}{poolHK}{pool}{stake}{avgs}{tot}{addrs_{rew}} =
+      & \rewardOnePool{pp}{R}{n}{\overline{N}}{poolHK}{pool}{stake}{tot}{addrs_{rew}} =
           (\var{rewards},~\var{unrealized})\\
```

Comment:
Since we are now moving all of `R-sum(f^)` to the treasury (see page 40 of the delegation design doc, the very end of 6.5.2), we no longer need to track the difference between `f` and `f^`. this simplifies things!

Neither `rewardOnePool` nor `reward` need to return `unrealized` anymore. This means that the reward pot is no longer needed! (see #399). In `createRUpd` we do **not** need to compute rp anymore. The treasury will now absorb these funds, so that we can define `∆t2` to be `R-(∑c)`.

Reply:
this is important since we cannot guarantee `sum(f)-sum(f^)` is positive. What we do know is that `R-sum(f^)` is positive.